docs/user-guide/authenticating-with-client-certificates.md

```diff
@@ -0,0 +1,82 @@
+# Authenticating with client certificates
+
+You can perform authentication with client certificates.
+
+If the keyring or a truststore contains at least one valid certificate authority (CA) other than the CA of API ML, it is possible to use the client certificates issued by this CA to authenticate to API ML. This feature is not enabled by default and must be configured.
+
+:::note
+There is a limitation with respect to ACF2 systems. If you would like to offer feedback using client
+certificate authentication, please create an issue in the Zowe [api-layer repository](https://github.com/zowe/api-layer).
+:::
+
+When providing credentials during a login request with a client certificate on the same login request, the credentials take precedence and the client certificate is ignored.
+
+## How the Gateway resolves authentication 
+
+When sending a request to a service with a client certificate, the Gateway performs the following process to resolve authentication:
+
+* The client calls the service endpoint through API ML Gateway with the client certificate.
+* The client certificate and private key are checked as a valid TLS client certificate against the trusted CAs of the Gateway.
+* The public part of the provided client certificate is checked against SAF. SAF subsequently returns a user ID that owns this certificate. Z Secure Services (ZSS)* provides this API for API ML. 
```

Comment:
@janan07 This needs to be updated to reflect that from Zowe version 2.14, ZSS is the default API that provides this mapping between the public part of the client certificate and SAF user Id, but this Zowe version introduces a secondary option which is an implementation within API ML that does not require ZSS. This needs to be explicitly set in Gateway's configuration in zowe.yaml

Reply:
I added some language. Please review.